### ISSUE TITLE
the data format for Chinese？

### ISSUE THREAD

**Issue by Chenweihua405** [salestock/fastText.py#129]
model = fasttext.skipgram('data.txt', 'model')；

I want to do the text classification for chinese ,I wonder the data format of data.txt ?

**Comment by spate141** [comment]
`skipgram` is for word2vec model, `supervised` is for classification!
For classification, `__label__LABEL1 sentence` is the format

**Comment by dcsan** [comment]
as long as you segment your text before input it should work. I've done this and using the pretrained language set for chinese, the classification is very impressive.

**Comment by gembin** [comment]
@dcsan which library you used for Chinese  segmentation?

**Comment by dcsan** [comment]
@gembin node-jieba

**Comment by qitong** [comment]
@dcsan how big is your training set and how many categories to classify?
This is my sample line (I use jieba to seg): "念得 我 各种 烦躁 … … __label__2" is that correct?
Do you know if there is a way to integrate pre-trained fasttext word embedding for the classification task?
Thank you.

**Comment by dcsan** [comment]
@qitong i think label has to be at the start of the line.

yes, I'm using their pretrained vectors. you just have to specify `-dim 300` on the various commands as that is how many dimensions the pretrained vectors are at - which is not the default I don't think.
